server.js

```diff
@@ -1,5 +1,32 @@
 const app = require('./app')
+const mongoose = require('mongoose')
+require('dotenv').config()
+require('colors')
 
-app.listen(3000, () => {
-  console.log("Server running. Use our API on port: 3000")
+const PORT = process.env.PORT || 3000
+const uriDb = process.env.DB_HOST
+
+mongoose.set('strictQuery', true)
+
+const connection = mongoose.connect(uriDb, {
+	useNewUrlParser: true,
+	useUnifiedTopology: true,
 })
+
+connection
+	.then(() => {
+		console.log('\nDatabase connection successful'.green)
+		app.listen(PORT, () => {
+			console.log(`Server running. Use our API on port: ${PORT}`.green)
+		})
+	})
+	.catch(err => {
+		console.log('\nDatabase not running\n'.red, err.toString())
+		process.exit(1)
+	})
+
+function signalHandler() {
+	mongoose.disconnect()
+	console.log('\nDatabase disconnected\n'.red)
+}
+process.on('SIGINT', signalHandler)
```

Comment:
czy to Ci dziala na pojedyncze `ctrl+c` w terminalu, w ktorym dziala proces appki, czy musisz robic podwojne `ctrl+c`? Jesli podwojne, to rzuc okiem tu: https://stackoverflow.com/questions/14372288/how-to-properly-handle-sigint-with-express-js